polyglot/piranha/test-resources/kt/random.kt

```diff
@@ -0,0 +1,71 @@
+/* 
+    Copyright (c) 2022 Uber Technologies, Inc.
+    
+     <p>Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file
+     except in compliance with the License. You may obtain a copy of the License at
+     <p>http://www.apache.org/licenses/LICENSE-2.0
+    
+     <p>Unless required by applicable law or agreed to in writing, software distributed under the
+     License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+     express or implied. See the License for the specific language governing permissions and
+     limitations under the License.
+    */
+    package com.uber.input
```

Comment:
Question: Where or in which test is this new file used? I don't see it referenced anywhere else, nor does it seem to match the usual input/expected format for test cases... the other 4 changed files in `test-resources/kt` don't have any new references to this file...

Reply:
oh jeez. This is my mistake. I was using this file for some debugging. I ll remove this.